_posts/2017-08-25-aug-25.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: "This Week In Erlang Aug 25"
+description: ""
+date: 2017-08-25
+tags: [erlang, news]
+comments: false
+share: true
+---
+
+Hello and welcome to "This week in Erlang"! As expected, there was a lot of action on the OTP side this week, and maybe a little too slow everywhere else. However, there was a piece of great news that came out this week. Read on!
```

Comment:
I found 2 misspelled words in this sentence: 
- OTP
	- *Suggested Replacements:* olp, ot, ptp, otc, opt, op, tp, opp, top
- Erlang